MachineLearning_Engine/Convert/ToPython.cs

```diff
@@ -67,6 +67,18 @@ public static PyObject ToPython(this LinearRegression model)
             return model.SkLearnModel;
         }
 
+        /***************************************************/
+
+        public static PyObject ToPython(this MinMaxScaler scaler)
+        {
+            return scaler.SkLearnScaler;
+        }
+
```

Comment:
Can you add the `/***************************************************/` separator here?